### PR TITLE
Take violations out of beta

### PR DESCRIPTION
**Pull request by ShridharGoel**
### Details

Take violations out of beta.

### Fixed Issues

$ https://github.com/Expensify/App/issues/44995
PROPOSAL: https://github.com/Expensify/App/issues/44995#issuecomment-2214925503

### Test steps

https://github.com/Expensify/App/issues/44995#issue-2396295781

### QA steps

https://github.com/Expensify/App/issues/44995#issue-2396295781

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/user-attachments/assets/c0537d56-b836-4d43-8b0a-acab5d382fd9

https://github.com/user-attachments/assets/1bdebbeb-6d26-4950-a9f3-6637c086d860

https://github.com/user-attachments/assets/09d3cea6-1a6b-4ac0-9f70-4d24b8a981ea

https://github.com/user-attachments/assets/865aef5a-4906-4d50-9722-4b6d0bc000d5

https://github.com/user-attachments/assets/ba0e510c-889f-408b-ab58-75f6731715e9

https://github.com/user-attachments/assets/8e615d6c-5a29-451c-9744-5a60cb0af0cb

https://github.com/user-attachments/assets/f691c4d0-8a45-49d0-9122-78f9ca999c00

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/user-attachments/assets/56d5e462-efff-4b5e-8bb2-10ff639f58e7

https://github.com/user-attachments/assets/8be6cc4e-2372-4020-a993-99e2b373681c

https://github.com/user-attachments/assets/90df0795-d934-4937-a1f1-83ddc589f0e9

https://github.com/user-attachments/assets/001cfad6-07ca-448f-89a2-e34113fbb4f7

https://github.com/user-attachments/assets/44618a99-51ac-45f7-9d8b-c95a95e11e0b

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/user-attachments/assets/653cd550-b46f-4cc3-883d-fd02e686add3

https://github.com/user-attachments/assets/66e2051b-f514-42f1-ab73-78a9b5477a37

https://github.com/user-attachments/assets/6273d089-3c59-47e7-9fcf-3a272ff82883

https://github.com/user-attachments/assets/bb5cbf0e-7fe7-4952-968e-229ce54757a8

https://github.com/user-attachments/assets/0a26e323-b73c-4d65-9e89-f7a3a87b7543

https://github.com/user-attachments/assets/5d2a585d-81bf-4420-8274-710189613544

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/user-attachments/assets/c705c4bd-ca2e-4e9e-8e61-3f1e0c3561c3

https://github.com/user-attachments/assets/683e048b-3ea5-4e86-aa1a-dcef9a463975

https://github.com/user-attachments/assets/52ef31e6-5de2-43b7-a6ab-2b0ffec76ddb

https://github.com/user-attachments/assets/c4dba3ee-6ed3-4191-a8de-76512d7734c8

https://github.com/user-attachments/assets/0bcf3e61-b93d-4fb1-b4c8-aa1769812e25

https://github.com/user-attachments/assets/931df4c6-debb-4f4d-8969-266ba7e44960

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/user-attachments/assets/ff7631e6-555c-471e-8a52-885fe6af404e

https://github.com/user-attachments/assets/6f2ebbce-06fa-4c4b-a9ba-8c7a94e59dac

https://github.com/user-attachments/assets/a8bd48d1-79f0-44e0-9e9b-72f590c82bf5

https://github.com/user-attachments/assets/ffe6c028-cf84-4962-99ac-f1a2ed362efe

https://github.com/user-attachments/assets/30d39fe4-21a6-411f-8fa4-4caaa0e943ff

https://github.com/user-attachments/assets/68d63ae1-a1bf-490d-a0b0-152b181ff0d3

https://github.com/user-attachments/assets/f1c16e9d-6e9a-4c40-8738-3af44f3a1935

https://github.com/user-attachments/assets/0891fd02-d323-46a5-b053-804cf2f51065

https://github.com/user-attachments/assets/5e2c4325-b863-44ea-a0b2-0693d15cb35c

https://github.com/user-attachments/assets/ce5500a6-33da-4cdf-a43b-06ac8256f193

https://github.com/user-attachments/assets/2852c2cf-f8ed-41ca-886c-83f951f4d701

https://github.com/user-attachments/assets/81f5b44e-fa9a-4a5f-9b26-494080554ded

https://github.com/user-attachments/assets/6f35712e-e768-492d-828e-6e231941a539

https://github.com/user-attachments/assets/d97b794b-2dbe-493d-bd3b-d8a90aa07bd5

https://github.com/user-attachments/assets/378ad6a5-3517-4a72-b7a4-8e636766d787

https://github.com/user-attachments/assets/d9f38f6e-45a8-4d65-8ee3-d84d9aaf3aaa

https://github.com/user-attachments/assets/fa58cb1f-0a19-4e86-9027-921bead25539

https://github.com/user-attachments/assets/23d4396d-225e-456e-b067-cf2352f4b9aa

Screenshot 2024-07-24 at 7 53 59 PM

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/user-attachments/assets/1ce784ff-a083-4a59-be07-e31a650fae25

https://github.com/user-attachments/assets/a04e1638-ce9a-4a17-b6d8-28a004159311

https://github.com/user-attachments/assets/72dec527-c76a-4c4d-80bd-a9982362028f

https://github.com/user-attachments/assets/3baaa68c-090c-49a4-967d-e2bb921668ed

https://github.com/user-attachments/assets/a5e33ee3-a174-408f-8b8d-186e00e82a83

https://github.com/user-attachments/assets/2035ba8a-9938-4c7d-9ee8-4da9d8da01c6

https://github.com/user-attachments/assets/34a02633-827d-445a-98b6-f79c2b99a231

https://github.com/user-attachments/assets/80ef38b9-4dd8-4898-87ad-ec5c33b09f1b

https://github.com/user-attachments/assets/a5464c71-e840-44d8-b2e1-686f7c11db68

</details>

### CODE REVIEW

File: tests/unit/SidebarOrderTest.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ import wrapOnyxWithWaitForBatchedUpdates from '../utils/wrapOnyxWithWaitForBatch
 
 // Be sure to include the mocked Permissions and Expensicons libraries as well as the usePermissions hook or else the beta tests won't work
 jest.mock('@libs/Permissions');
-jest.mock('@hooks/usePermissions.ts');
```
Comment (getusha) [inline tests/unit/SidebarOrderTest.ts:15]: Why are we removing this? is this only needed for violations beta?

File: src/libs/actions/IOU.ts [hunk 2]
```diff
@@ -5504,14 +5493,11 @@ function cleanUpMoneyRequest(transactionID: string, reportAction: OnyxTypes.Repo
         });
     }
 
-    // added the operation to delete associated transaction violations
```
Comment (getusha) [inline src/libs/actions/IOU.ts:5507]: Lets keep this comment

File: tests/unit/SidebarOrderTest.ts [hunk 3]
```diff
@@ -12,7 +12,6 @@ import wrapOnyxWithWaitForBatchedUpdates from '../utils/wrapOnyxWithWaitForBatch
 
 // Be sure to include the mocked Permissions and Expensicons libraries as well as the usePermissions hook or else the beta tests won't work
```
Comment (getusha) [inline tests/unit/SidebarOrderTest.ts:13]:
```suggestion
// Be sure to include the mocked Permissions and Expensicons libraries or else the beta tests won't work
```

File: tests/unit/SidebarTest.ts [hunk 4]
```diff
@@ -12,7 +12,6 @@ import wrapOnyxWithWaitForBatchedUpdates from '../utils/wrapOnyxWithWaitForBatch
 
 // Be sure to include the mocked Permissions and Expensicons libraries as well as the usePermissions hook or else the beta tests won't work
```
Comment (getusha) [inline tests/unit/SidebarTest.ts:13]:
```suggestion
// Be sure to include the mocked Permissions and Expensicons libraries or else the beta tests won't work
```

**Comment by getusha** [comment]
@ShridharGoel could you structure the videos with their respective title?

**Comment by getusha** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [ ] iOS: Native
    - [ ] iOS: mWeb Safari
    - [ ] MacOS: Chrome / Safari
    - [ ] MacOS: Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [ ] I verified that all the inputs inside a form are aligned with each other.
    - [ ] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos

### Missing tag and missing category violations

## Manual Request

<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

## Scan Request

<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

### Category out of policy, tag out of policy violation, expense over limit, max age, over category limit, receipt required

## Manual Request

<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

## Scan Request

<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

## Distance Request

<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

### Tag violations on workspace with Independent Tags (multi-level tags)

## Manual Request

<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

### Tag violations on workspace with Dependent Tags (multi-level tags)

## Manual Request

<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

### Receipt Audit Scan

<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

### Receipt Audit - Receipt Not Verified

<details>
<summary>Android: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: Native</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<!-- add screenshots or videos here -->

</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->

</details>

**Comment by cead22** [comment]
- @getusha I know there's a lot to test here, but I'm curious if you have an ETA for when you'll have the reviewer checklist ready
- @ShridharGoel there's a small lint error that you need to fix before we can merge

**Comment by getusha** [comment]
>@getusha I know there's a lot to test here, but I'm curious if you have an ETA for when you'll have the reviewer checklist ready

I should be able to complete it by EOD tomorrow, at the latest.

**Comment by cead22** [comment]
Awesome, thanks!

**Comment by getusha** [comment]
I think you missed this @ShridharGoel https://github.com/Expensify/App/pull/45760#issuecomment-2257475827

**Comment by getusha** [comment]
> Verify you see the Missing Category red text under the category row

Category is selected by default, if you leave it blank. which fails ^

1. Have one category under the recently section
2. Submit expense without filling Category and Tags

https://github.com/user-attachments/assets/cc558885-8b9f-47e7-ab68-c24375c78863

Could you reproduce this? @ShridharGoel

**Comment by getusha** [comment]
> @getusha I don't think so. Is it happening each time?

Yes, could you try following the steps here https://github.com/Expensify/App/pull/45760#issuecomment-2262906520?

**Comment by getusha** [comment]
>Switch back to new dot, and refresh the page
 Verify you see Category no longer valid violation
 Verify you see Tag no longer valid violation
 
 This is failing for Tag on android native after disabling the selected tag from old dot

https://github.com/user-attachments/assets/feb1f96e-2de4-49f0-96a1-34f6d4485136

**Comment by cead22** [comment]
If you're getting a category applied automatically it's probably due to a feature we call "implicit autocategorization", and you can work around it by using a new random merchant name every time.

What this feature does is, when you add a category to a transaction with a specific merchant, next time you create another transaction with the same merchant, we'll assume the category is the same on you used the previous time for the same merchant.

That said, if a category is applied, you can de-select it and then confirm the "missing category" violation is present.

**Comment by cead22** [comment]
What's the ETA on getting all the videos uploaded and the review complete? Also, please follow the checklist for the console warnings & errors

> * [x]  I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)

**Comment by getusha** [comment]
>What's the ETA on getting all the videos uploaded and the review complete?

I am on it, tests are failing for me, started a thread https://expensify.slack.com/archives/C01GTK53T8Q/p1722592136673719

**Comment by getusha** [comment]
`BUG:` `Tag no longer valid` appears on optional multilevel tag after changing a required tag

1. Enable multiple levels of tag (oldDot)
2. Import the tags from https://github.com/Expensify/App/issues/37117#issuecomment-2177239758 (oldDot)
3. Mark the first two tags not required (oldDot)
4. Submit expense > Enter amount > Enter merchant > Submit
5. Open the expense preview
6. Select a project from the project multi-tag

https://github.com/user-attachments/assets/67fa07af-feee-41f5-ba07-39f10c385840

**Review by getusha — APPROVED** [review]

**Comment by cead22** [comment]
> * [x]  I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)

@getusha can you report these and link the slack convo here please? Thanks

**Review by cead22 — APPROVED** [review]

**Comment by kavimuru** [comment]
@cead22 Should we run all the steps in the linked PR? We do most of them in regular regression run

**Comment by getusha** [comment]
>@getusha can you report these and link the slack convo here please? Thanks

@cead22 I noticed an error that's only visible in Dev. Should I raise a PR to fix it? Also, do we need to report the warnings on native platforms?

**Comment by cead22** [comment]
> @cead22 I noticed an error that's only visible in Dev. Should I raise a PR to fix it? Also, do we need to report the warnings on native platforms?

@getusha Yes and yes please

**Pull request merged**